.travis.yml

```diff
@@ -0,0 +1,29 @@
+language: python
+dist: xenial
```

Comment:
we can use the latest distribution `bionic`(refer doc [here](https://docs.travis-ci.com/user/reference/overview/))

Reply:
Cool. Will have it updated in this #3 PR.